src/assets/style.css

```diff
@@ -454,3 +496,193 @@ section#contact > form textarea {
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 20px;
+    color: #fff;
+  }
+
+  /* HEADER ENDED */
+
+  /* HEADLINE SECTION STARTED */
+
+  section#headline {
+    row-gap: 24px;
+    background-image: url("Illustration-header.svg");
+    background-size: contain;
+    background-position-x: 118%;
+    background-position-y: center;
+  }
+
+  section#headline > .info {
+    width: 490px;
+    padding: 0;
+    display: flex;
+    flex-direction: column;
+    row-gap: 24px;
+  }
+
+  section#contact > h2 {
+    width: 455px;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  section#headline > .info > h2 {
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 82px;
+    color: #f55800;
+    padding: 0;
+  }
+
+  section#headline > .info > h3 {
+    font-size: 32px;
+    line-height: 44px;
+    color: #fff;
+  }
+
+  section#headline > .info > p {
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  section#headline > ul.social_media {
+    order: -1;
+    flex-direction: column;
+    margin-right: 87px;
+    row-gap: 23px;
+  }
+
+  /* HEADLINE SECTION ENDED */
+
+  /* WORKS SECTION STARTED */
+
+  section#works {
+    margin-top: 130px;
+    margin-bottom: 130px;
+  }
+
+  section#works > .title {
+    display: flex;
+    align-items: center;
+    column-gap: 24px;
+  }
+
+  section#works > .title > h2 {
+    white-space: nowrap;
+  }
+
+  section#works > .title > .line {
+    width: 100%;
+    height: 1px;
+    transform: translateY(-10px);
+  }
+
+  section#works > .cards {
+    margin-top: 114px;
+    display: grid;
+    row-gap: 24px;
+  }
+
+  section#works > .cards > .specialCard {
+    flex-direction: row;
+    flex-wrap: wrap;
+    column-gap: 24px;
+    justify-content: center;
+  }
+
+  section#works > .cards > .specialCard > img {
+    width: 60%;
+  }
+
+  section#works > .cards > .specialCard > .description {
+    width: 400px;
+    display: flex;
+    flex-direction: column;
+    row-gap: 24px;
+  }
+
+  section#works > .cards > .otherCards {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    row-gap: 36px;
+    justify-content: space-around;
+  }
+
+  section#works > .cards > .otherCards > .card {
+    width: 30%;
+    min-width: 280px;
+  }
+
+  /* WORKS SECTION ENDED */
+
+  /* ABOUT SECTION STARTED */
+
+  section#about {
+    margin-top: 130px;
+  }
+
+  section#about > .info {
+    background-image: url("illustration-about\ me.svg");
+    background-repeat: no-repeat;
+    background-position: center right;
+  }
+
+  section#about > .info > .content {
+    width: 500px;
+  }
+
+  section#about > .info > .content > button {
+    margin-bottom: 130px;
+  }
+
+  section#about > .hardSkills {
+    flex-direction: row;
+    justify-content: space-around;
+    align-items: flex-start;
+  }
+
+  section#about > .hardSkills > ul {
+    align-items: flex-start;
+  }
+
+  section#contact {
```

Comment:
- [ ] Kindly make sure your contact section look like the Figma design in case of proportions :recycle: 

- Your design:
![Screenshot from 2023-01-27 15-01-17](https://user-images.githubusercontent.com/97834160/215093644-47b55f56-e059-4167-bc31-eb9b0198fb2b.png)



-Figma design:

![Screenshot from 2023-01-27 15-02-40](https://user-images.githubusercontent.com/97834160/215093880-552b28f1-acda-44e2-aee3-dfe4253a3b79.png)